ajax/impact.php

```diff
@@ -120,7 +120,7 @@
         }
 
        // Decode data (should be json)
-        $data = Toolbox::jsonDecode($_UPOST['impacts'], true);
+        $data = Toolbox::jsonDecode($_POST['impacts'], true);
```

Comment:
Are you sure for this one? I had in mind JSON causes issues when used from `$_POST`

Reply:
I have modified the `Toolbox::jsonDecode()` to ensure sanitization process is correctly reverted, and I added a test case for this.

Reply:
ok :)